Monika After Story/game/script-food.rpy

```diff
@@ -0,0 +1,153 @@
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_imeating",
+                    category=['you'],prompt="I'm Eating/Drinking",pool=True,unlocked=True,
+                    mas_getEV('monika_imeating').shown_count))
+if seen_event(monika_imeating) = False:
+    label monika_imeating:
+        m 1eua "You know, although I don't get hungry, I still miss enjoying good food."
+        m 4eub "Like Natsuki's cupcakes."
+        m 4eua "For a bunch of lines of code, they were pretty tasty."
+        m 3hua "Are you eating anything right now, [player]?"
+
+
+default persistent_.mas_food_database = {}
+
+default persistent_.mas_food_current = None
+
+init 1 python in mas_food:
+    food_db = dict()
+
+    TYPE_FOOD = 0
+    TYPE_DRINK = 1
+
+    FOOD_X = 680
+    FOOD_Y = 40
+    FOOD_W = 560
+    FOOD_H = 640
+    FOOD_XALIGN = -0.05
+    FOOD_AREA = (FOOD_X, FOOD_Y, FOOD_W, FOOD_H)
+    FOOD_RETURN = "Nothing"
+
+
+    def getFoodType(food_label):
```

Comment:
Where are you planning to use this?

Reply:
If you're talking about the "Nothing", it was for an exit from the menu.
If you meant "def getFoodType" it was whether or not it was a food or drink item.

Reply:
Alright, but where would you use the info of it being a food/drink item?

Reply:
You have a good point

Reply:
If you take Rai's suggestion and split into two topics (and use string categories), you can just populate both menus via a for loop using the condition `if 'food' in ev.category` (for food) and `if 'drink' in ev.category` (for drink).